priv/repo/data/courses.json

```diff
@@ -1767,6 +1767,19 @@
     "tool_names": [],
     "fundamentals_names": []
   },
+  {
+    "name": "Aprenda com quem não sabe",
+    "youtube_course_id": "PLr4c053wuXU-YY4GmqUM3y_uauspsmO0O",
+    "type": "playlist",
+    "locale": "pt",
+    "image_url": "https://i.ytimg.com/vi/XHxRv0f2MEY/hqdefault.jpg?sqp=-oaymwEbCKgBEF5IVfKriqkDDggBFQAAiEIYAXABwAEG&rs=AOn4CLC4xmzsMXWPUJKS88lT_4tDgsqalw",
+    "published_at": "2021-12-06T00:00:00Z",
+    "youtube_channel_id": "UCoMS25HuclMfa6IQJNcvh2w",
```

Comment:
You need to add this channel to `priv/repo/data/channels.json`:

```json
  {
    "name": "queroser.ninja",
    "image_url": "https://yt3.ggpht.com/ytc/AIdro_nlDClX-gQnjUYVOds8SZTFVUQVGTNu6V1SnlwJp--SNiU=s88-c-k-c0x00ffffff-no-rj",
    "youtube_channel_id": "UCoMS25HuclMfa6IQJNcvh2w"
  }
```

Reply:
Ok